### PR TITLE
modify revue page

### PR DESCRIPTION
**Pull request by nacimgoura**

### CODE REVIEW

File: lib/components/FullTextHolding.js [hunk 1]
```diff
@@ -11,38 +11,39 @@ export const FullTextHolding = ({
...
+            {coverage ? (
+                <div className="coverage">
+                    {coverage
+                        .map(
+                            coverage =>
+                                `${text.date(coverage.start)} - ${text.date(
+                                    coverage.end,
+                                )}`,
+                        )
+                        .join(', ')}
```
Comment (RobinBressan) [inline lib/components/FullTextHolding.js]: you don't need to display the embargo?

File: lib/components/PublicationRecord.js [hunk 2]
```diff
@@ -7,6 +7,16 @@ import DL from './DL';
 import BibButton from './BibButton';
 import translate from '../higherOrderComponents/translate';
 
+// use MouseEvent instead of Jquery
+const simulateClick = elem => {
+    const evt = new MouseEvent('click', {
```
Comment (RobinBressan) [inline lib/components/PublicationRecord.js]: are you sure this won't mess up with react synthetic event?

File: lib/components/PublicationRecord.js [hunk 3]
```diff
@@ -36,17 +51,38 @@ class PublicationRecord extends Component {
...
+                        >
+                            {`${id}. ${title} ${titleType}`}
+                        </a>
+                    ) : (
+                        <a
+                            className="fetch-link resource-opener"
+                            href="#"
+                            onClick={() => showResource(id)}
+                        >
+                            {`${id}. ${title} ${titleType}`}
```
Comment (RobinBressan) [inline lib/components/PublicationRecord.js]:
```suggestion
                            {id}. {title} {titleType}
```

File: lib/components/FullTextHolding.js (hunk 1 above)
Comment (nacimgoura) [reply lib/components/FullTextHolding.js]: According to the mock-up, we're not supposed to display it anymore.

**Pull request merged**